File: Programmers/JAVA/lv1/lv1_두 정수 사이의 합.java
https://programmers.co.kr/learn/courses/30/lessons/12912

class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        long sum = 0;
        int tmp = 0;
        
        if(b < a){
            tmp = a;
            a = b;
            b = tmp;
        }
        
        for(int i = a; i <= b; i++){
            sum += i;
        }
        answer = sum;
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_두 정수 사이의 합.cs
https://programmers.co.kr/learn/courses/30/lessons/12912?language=csharp

public class Solution
{
    public long solution(int a, int b)
    {
        long answer = 0;
        int min = (a < b) ? a : b;
        int max = (a < b) ? b : a;
        for (int i = min; i <= max; i++)
        {
            answer += i;
        }
        return answer;
    }
}